msal4j-sdk/src/main/java9/com/microsoft/aad/msal4j/HttpHeaders.java

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@siddhijain @Avery-Dunn - please try to respond to community contributions. This looks ok to me, although it's missing tests. What is the context here? I don't see any feedback on this PR.

Reply:
As it seems this PR won't be merged, is there any fix you'd propose?

Reply:
@RJPG-TK For the time being, please continue using the workaround you provided in your message above, as we find a solution to the issue.